articles/client-platforms/angular2/07-authorization.md

```diff
@@ -7,7 +7,7 @@ description: This tutorial will show you how assign roles to your users, and use
   link: 'https://github.com/auth0-samples/auth0-angularjs2-systemjs-sample/tree/master/07-Authorization',
 }) %>
 
-<%= include('../_includes/_authorization-introduction', { ruleslink: '/quickstart/spa/angular2/07-rules' }) %>
+<%= include('../_includes/_authorization-introduction', { ruleslink: '/quickstart/spa/angular2/06-rules' }) %>
```

Comment:
It's used to point to the rules tutorial [here](https://github.com/auth0/docs/blob/master/articles/client-platforms/_includes/_authorization-introduction.md), so it's ok without md extension...